va/validation-authority.go

```diff
@@ -195,6 +240,24 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 	return challenge, err
 }
 
+func (va ValidationAuthorityImpl) getAddrs(hostname string) (addrs []net.IP, problem *core.ProblemDetails) {
```

Comment:
Please do not put this between the validation functions. Put it at the top or the bottom.